src/views/product-landing/server.ts

```diff
@@ -66,7 +66,7 @@ async function generateStaticProps({
       headings: extractHeadings(content),
       breadcrumbLinks: [
         { title: 'Developer', url: '/' },
-        { title: product.name, url: `/${product.slug}` },
+        { title: product.name, url: `/${product.slug}`, isCurrentPage: true },
```

Comment:
@ashleemboyer Update since last review: in this server file and a couple others, setting `isCurrentPage` on the correct breadcrumb as needed.

I realized as I was doing this we could likely simplify the logic here: if we can trust our breadcrumb bar generation, then the last item in a breadcrumb bar should be marked as current page (which we could do easily on the front end, no need to pass `isCurrentPage` through). Make a task for that as it felt outside the scope of this PR: 🎟️ [[breadcrumb-bar] simplify isCurrentPage logic - always highlight last item, rather than based on href](https://app.asana.com/0/1202097197789424/1202354347457831/f)